naming/src/main/java/com/alibaba/nacos/naming/misc/NamingExecuteTaskDispatcher.java

```diff
@@ -19,13 +19,16 @@
 import com.alibaba.nacos.common.task.AbstractExecuteTask;
 import com.alibaba.nacos.common.task.engine.NacosExecuteTaskExecuteEngine;
 import com.alibaba.nacos.sys.env.EnvUtil;
+import org.springframework.beans.factory.DisposableBean;
+import org.springframework.stereotype.Component;
 
 /**
  * Naming execute task dispatcher.
  *
  * @author xiweng.yy
  */
-public class NamingExecuteTaskDispatcher {
+@Component
```

Comment:
Not change it as spring bean, I means the old implementation `getExecuteEngine` change to `destory` and called in the code which calls `getExecuteEngine`